<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>281</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
background-color: rgb(255,255,255);
border-radius:15px;
}</string>
   </property>
  </widget>
  <widget class="QSlider" name="PlayTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>505</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
	background-color:rgb(255,255,255);
	background-size:20% 20%;
    border-radius: 4px; /* 设置滑块的边框圆角 */
	box-shadow:5px 5px 5px rgba(0,0,0,1);
}
QSlider::handle:horizontal:hover {
	background-color:rgb(212,212,212);
	background-size:20% 20%;
    border-radius: 4px; /* 设置滑块的边框圆角 */
	box-shadow:5px 5px 5px rgba(0,0,0,1);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="endLable">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>485</y>
     <width>31</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="startLable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>485</y>
     <width>31</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255,255,255);
	icon-size:25px;
	border-radius:15px;
}
QPushButton:hover{
	background-color:rgb(212,212,212);
}
QPushButton:pressed{
	background-color:rgb(199,199,199);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255,255,255);
	icon-size:25px;
	border-radius:15px;
}
QPushButton:hover{
	background-color:rgb(212,212,212);
}
QPushButton:pressed{
	background-color:rgb(199,199,199);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:red;
border-radius:5px;
}
QPushButton:hover{
	background-color:rgb(205, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:rgb(65, 205, 82);
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgb(65,155,82)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="wg_bg" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>300</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:rgb(240, 240, 240);
	border-radius:15px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>540</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	icon-size:45px 45px;
	background-color:rgb(255,255,255);
	border-radius: 15px;
	box-shadow:55px 55px 55px rgb(0,0,0,1)
}
QPushButton:hover{
	background-color:rgb(212,212,212);
}
QPushButton:pressed{
	background-color:rgb(199,199,199);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="options">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color:rgb(0, 103, 232);
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgb(0, 20, 122)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>wg_bg</zorder>
  <zorder>listWidget</zorder>
  <zorder>PlayTime</zorder>
  <zorder>endLable</zorder>
  <zorder>startLable</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>close</zorder>
  <zorder>min</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>options</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
